src/__tests__/index.test.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import TestRenderer from 'react-test-renderer';
+import Index from '../pages/index';
+import 'jsdom-global/register';
+
+
```

Comment:
```suggestion
```

This should be in the eslint([no-multiple-empty-lines](https://eslint.org/docs/rules/no-multiple-empty-lines)) but let's remove unnecessary spacing for consistency

Reply:
I already add the rule to my linter, I'll commit the changes.